### ISSUE TITLE
Character addition request.

### ISSUE THREAD

**Issue by bedriczwaleta** [microsoft/cascadia-code#419]
This font does not contain Arabic (namely Pashto).
The bitcoin sign is missing, ḂḃḊḋḞḟṀṁṖṗṠṡṪṫ is missing as well.
No more to explain, I'm bored.

**Comment by aaronbell** [comment]
Sorry you're bored. 

We already have a bug related to [Arabic](https://github.com/microsoft/cascadia-code/issues/84).

The bitcoin glyph is one I will consider adding, as are the dot above glyphs, but you can work around those for now by using the dotabove combining mark (`̇`, `uni0307`).

**Comment by bedriczwaleta** [comment]
But 84 is not as long, and too unexplained, in its early development.

Also U+0659 is required in the distant future... I do think. 84 is just nothing.

**Comment by aaronbell** [comment]
Sorry, your comment makes no sense.

**Comment by bedriczwaleta** [comment]
What does that means? I plan Bitcoin, Pashto, Hebrew but not Yiddish, Thai, to make Cascadia multilingual. You think that's misinformation.

**Comment by yg8ijvjvjv** [comment]
ḂḃḊḋḞḟṀṁṖṗṠṡṪṫ are in [Subset1](https://unicodesubsets.miraheze.org/wiki/Subset1) (678 characters), inherited from [ISO 8859-14](https://unicodesubsets.miraheze.org/wiki/ISO_8859-14). Subset1 has 21 characters not in WGL4. I recommend monospaced fonts contain Subset1, despite bedriczwaleta not recommending it. [Subset2](https://unicodesubsets.miraheze.org/wiki/Subset2) (1193 characters) could be the next step. Bedriczwaleta says that a  monospaced Unicode-compilant font can have as little as 6125 characters.

**Comment by aaronbell** [comment]
Thanks for helping @yg8ijvjvjv, though based on researching those specific glyphs, the request is likely more in relation to Hebrew romanization than alignment with ISO standards.

Given the above, let me rephrase the feature request:

> I would like to use Cascadia Code for Arabic (Pashto) and Hebrew applications but noticed that the font doesn’t support Arabic. Could that be added? It would also be useful to add `ḂḃḊḋḞḟṀṁṖṗṠṡṪṫ`, as those glyphs are used for [Hebrew romanization](https://en.m.wikipedia.org/wiki/Dot_(diacritic)). And the Bitcoin symbol too, please. Thank you!

As I stated previously, issue #84 covers the addition of Arabic, and I will consider the other requests. Thanks for submitting an issue.

**Comment by yg8ijvjvjv** [comment]
It's not an "alignment with ISO standards" it is compatibility. If the font contains all ISO 8859-14 characters, it can be used to simulate ISO 8859-14 environments. And after Subset1, when developing Subset2 in a font, it makes it compatible with documents relying on Subset2 characters. For instance, [DMCA Sans Serif](https://typedesign.netlify.app/dmcasansserif.html) 8.1 has Subset2 character set. Also ḂḃḊḋḞḟṀṁṖṗṠṡṪṫ, aren't used for Hebrew romanization, it's for old Irish instead.

**Comment by yg8ijvjvjv** [comment]
@aaronbell You need to edit your comment, it's misinformation. B, D, F, M, P, S, T dot not Hebrew romanisation, as above mentioned.

**Comment by yg8ijvjvjv** [comment]
There are 17 missing Subset1 characters, namely ḂḃḊḋḞḟṀṁṖṗṠṡṪṫ�. The last 3 are F001, F002, FFFD. The F001—F002 are required by WGL4 and should be duplicates of FB01—FB02. The FFFD is inherited from VSECS and is used in some environments for some errors. And extremely many missing Subset2 characters. 

Bedriczwaleta thinks that Cascadia Code/Code PL/Mono/Mono PL will not have Subset1 nor Subset2 except for BDFMPST dot.

**Comment by aaronbell** [comment]
> In the ISO 259 Romanization of Hebrew, the overdot is used to transcribe the dagesh: ⟨ḃ ḋ ġ ḣ ṁ ṅ ṙ ṡ ṥ ṧ ṩ ṫ⟩; ⟨ẇ⟩ transcribes the shuruk.

🤷‍♂️

**Comment by bedriczwaleta** [comment]
No. That is old Irish.

**Comment by aaronbell** [comment]
If you would like to file a bug to support ISO subset1 and 2 please do so. Thanks.

**Comment by yg8ijvjvjv** [comment]
> Traditional Irish typography, where the dot denotes lenition, and is called a ponc séimhithe or buailte "dot of lenition": ḃ ċ ḋ ḟ ġ ṁ ṗ ṡ ṫ.

Note that the ċ and ġ are already existent as in Maltese (included in ISO 8859-3 and CP853 and in turn in WGL4).

Subset2 is a general-purpose character set. It is designed to merge SECS, W1G, MES-2, IPA-MINI and the character U+203F. So it has Greek Polytonic and extended Cyrillic from MES-2, and extremely many math symbols inherited from SECS and such, making it general-purpose. The next step would be [Subset3](https://unicodesubsets.miraheze.org/wiki/Subset3), 2823 characters. It was significant when DMCA Sans Serif 9.0 was released, being intended to at least contain Subset3, it was released with [Subset3+](https://unicodesubsets.miraheze.org/wiki/Subset3%2B) with 3309 characters. See [https://unicodesubsets.miraheze.org/wiki/Category:All_character_set_pages] for all character set pages.

**Comment by bedriczwaleta** [comment]
No. Just no. This is Pashto

ابپتټثجځچڅحخدډذرړزژږسشښصضطظعغفقکګلمنڼوهیېيۍئےءَُِٙـ،؛؟۹۸۷۶۵۴۳۲۱۰, ﴾﴿ﷲ﷼, but also you need more Presntation forms A and B. and 
(sorry if I put duplicates in the diacs, Ẓẓ is needed as well.)

**Comment by aaronbell** [comment]
I have created three new issues to better track the separate character addition requests by @bedriczwaleta. As such, am closing this thread. Thanks.

**Comment by bedriczwaleta** [comment]
If you want to know, Subset1 was made by @yg8ijvjvjv for his monospaced fonts as a requirement

**Comment by yg8ijvjvjv** [comment]
> If you want to know, Subset1 was made by @yg8ijvjvjv for his monospaced fonts as a requirement

However, all of the fonts in this repository are proportional.
